get-substrate.sh

```diff
@@ -78,11 +78,17 @@ if [[ "$1" == "--fast" ]]; then
 	echo "You can install manually by cloning the https://github.com/paritytech/substrate repo,"
 	echo "and using cargo to install 'substrate' and 'subkey' from the repo path."
 else 
+	BOLD_RED='\033[1;31m'
+	NO_COLOR='\033[0m'
+	ERROR_MSG="\n\n${BOLD_RED}Build failed:${NO_COLOR} This can be due to latest rust nightly \
+breaking substrate, for more precise guidelines go to \
```

Comment:
Actually, we could run ` rustc --version` and ` rustc +nightly --version` [here](https://github.com/paritytech/substrate/blob/master/.gitlab-ci.yml#L417), save it to build artifacts and/or binary metadata to query in different places.
This script, for instance, could at least warn about the mismatch of the rust versions.